spyder/plugins/updatemanager/workers.py

```diff
@@ -219,35 +179,31 @@ def get_asset_info(release_info: dict) -> AssetInfo | None:
     else:
         update_type = UpdateType.Micro
 
-    if updater or update_type != UpdateType.Major:
-        ext = ".zip"
+    if updater:
+        filename = "spyder-updater.zip"
+    elif update_type != UpdateType.Major:
+        filename = "spyder-conda-lock.zip"
     else:
-        ext = platform.machine().lower().replace("amd64", "x86_64")
+        mach = platform.machine().lower().replace("amd64", "x86_64")
```

Comment:
```suggestion
        machine = platform.machine().lower().replace("amd64", "x86_64")
```

Let's use `machine` for clarity.